capellambse_context_diagrams/_elkjs.py

```diff
@@ -42,11 +42,11 @@
 NODE_HOME = Path(capellambse.dirs.user_cache_dir, "elkjs", "node_modules")
 PATH_TO_ELK_JS = Path(__file__).parent / "elk.js"
 REQUIRED_NPM_PKG_VERSIONS: t.Dict[str, str] = {
-    "elkjs": "0.8.1",
+    "elkjs": "0.8.2",
 }
 """npm package names and versions required by this Python module."""
 
-LayoutOptions = dict[str, t.Union[str, int, float]]
+LayoutOptions = cabc.MutableMapping[str, t.Union[str, int, float]]
 LAYOUT_OPTIONS: LayoutOptions = {
```

Comment:
If they're gonna be global, they should be immutable, i.e. `cabc.Mapping`. Then we get mypy errors when we accidentally do change them.